app/main.py

```diff
@@ -1,2 +1,59 @@
+from typing import Any, Hashable
+
+
 class Dictionary:
-    pass
+    class Node:
+        def __init__(self, key: Hashable, value: Any) -> None:
+            self.key = key
+            self.value = value
+            self.hash = hash(key)
+
+    def __init__(self, initial_capacity: int = 8) -> None:
+        self.capacity = initial_capacity
+        self.size = 0
+        self.load_factor = 2 / 3
```

Comment:
The load factor is set to 2/3, which is a common choice, but ensure that this is the intended value according to the task requirements. If the task specifies a different load factor, it should be adjusted accordingly.